deployment/flatpak/about.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Flatpak
+subsection: flatpak
+section: deployment
+description: >
+	Flatpak (formerly xdg-app) is a software utility for software deployment, package 	management and application virutalization. It provides a sandbox environment in which users 	can run applications in isolation from the rest of the system.
+---
+
+# {{page.title}}
+
+[Flatpak](https://flatpak.org/) is a software utility for software deployment, package management and application virtualization. It provides a sandbox environment in which users can run applications in isolation from the rest of the system.
+
+
+
+## Advantages of Using Flatpak
+
+* Flatpaks are distro agnostic. Developers can distribute their application on every Linux distribution simultaneously with ease.
+* Flatpaks provide consistent environments. Developers can develop and test applications in environments that are identical to the one users have.
+* Flatpak's build tools are simple and very easy to use. A link to the full documentation can be found in the "Further Reading" section below.
+* Flatpak runtimes provide stable sets of dependencies that can be used by applications.
+* Dependencies that aren't in a runtime can be bundled as part of an application. Versions of the dependencies that are used can be fixed and patched.
```

Comment:
This seems to go against principles that are customary to Fedora. Is this also the case on Fedora?
Keep in mind this should be Fedora specific. General comparisons can be linked to.

Reply:
> This seems to go against principles that are customary to Fedora.

Can you elaborate please?

Reply:
Yes, on IRC.